### PR TITLE
TTT-7: Test for player that has no cells of their symbol.

### PR DESCRIPTION
**Pull request by billlanyon**

### CODE REVIEW

File: test_ttt_game.py [hunk 1]
```diff
@@ -80,3 +80,11 @@ def test_11_is_any_row_complete_identifies_complete_row_for_wrong_player():
                [' ', ' ', ' '],
                [' ', ' ', ' ']]
     assert not g.is_any_row_complete('O')
+
+
+def test_12_is_any_row_complete_identifies_complete_row_for_wrong_symbols():
+    g = Game()
```
Comment (BasilBibi) [inline test_ttt_game.py]: This name is a bit confusing.

**Review by BasilBibi — CHANGES_REQUESTED** [review]
Please change the name of the test is is a bit confusing.

**Comment by billlanyon** [comment]
Naming this test is testing...

**Review by BasilBibi — APPROVED** [review]
Cool!

**Pull request merged**